Enigmatry.Entry.AspNetCore.Authorization/Attributes/UserHasRoleAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Enigmatry.Entry.AspNetCore.Filters;
+using Microsoft.ApplicationInsights.AspNetCore.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Logging;
+
+namespace Enigmatry.Entry.AspNetCore.Authorization.Attributes;
+public sealed class UserHasRoleAttribute : AuthorizeAttribute, IAuthorizationFilter
+{
+    public string Role { get; }
+
+    public UserHasRoleAttribute(string role) : base(PolicyNames.UserHasRole)
```

Comment:
I would suggest to have an additional constructor that accepts string[] roles.
This means user is allowed to access if it has any of the passed in roles.